products/extensions/b2b-components/employee-management/README.md

```diff
@@ -0,0 +1,3 @@
+# Employee Management
+
+A feature of the B2B components, including everything regarding employees, roles and permissions.
```

Comment:
I think "everything" is a little bit a fill word. I would recommend: "including employee, role and permission management"